Makefile

```diff
@@ -19,7 +19,10 @@ install-rust-coverage:
 .PHONY: build-dev
 build-dev:
 	@rm -f python/pydantic_core/*.so
-	maturin develop
+	cargo build --features extension-module
+	@rm -f target/debug/lib_pydantic_core.d
+	@rm -f target/debug/lib_pydantic_core.rlib
+	@mv target/debug/lib_pydantic_core.* python/pydantic_core/_pydantic_core.so
```

Comment:
Is this related to this PR or did it sneak in?

Reply:
This helps a lot when build fails.

Reply:
actually, it's required.